Emby.Server.Implementations/IO/ManagedFileSystem.cs

```diff
@@ -92,20 +73,25 @@ public virtual string ResolveShortcut(string filename)
             var extension = Path.GetExtension(filename);
             var handler = _shortcutHandlers.FirstOrDefault(i => string.Equals(extension, i.Extension, StringComparison.OrdinalIgnoreCase));
 
-            if (handler != null)
-            {
-                return handler.Resolve(filename);
-            }
-
-            return null;
+            return handler?.Resolve(filename);
         }
 
         public virtual string MakeAbsolutePath(string folderPath, string filePath)
         {
-            if (string.IsNullOrWhiteSpace(filePath)) return filePath;
+            if (string.IsNullOrWhiteSpace(filePath))
+            {
+                return filePath;
+            }
+
+            if (filePath.Contains("://"))
+            {
+                return filePath; // stream
+            }
 
-            if (filePath.Contains(@"://")) return filePath; //stream
-            if (filePath.Length > 3 && filePath[1] == ':' && filePath[2] == '/') return filePath; //absolute local path
+            if (filePath.Length > 3 && filePath[1] == ':' && filePath[2] == '/')
```

Comment:
this seems a Windows-only check, isn't there some standard mechanism in dotnet to check if given path is an absolute one?..

Reply:
There is in the new version of .Net core, not in the one we are using.